model/src/main/proto/exploration.proto

```diff
@@ -0,0 +1,145 @@
+syntax = "proto3";
+
+import "subtitled_html.proto";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Data for a single exploration.
+// Maps from: data/src/main/java/org/oppia/data/backends/gae/model/GaeExploration.kt
+message Exploration {
+  repeated State states = 1;
+  repeated ParamChange param_changes = 2;
+  repeated ParamSpec param_specs = 3;
+  string initStateName = 4;
```

Comment:
For consistency here & elsewhere: let's keep with snake case, e.g. ``init_state_name``

Reply:
Done

Reply:
Done